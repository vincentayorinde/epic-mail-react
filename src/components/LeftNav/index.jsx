import React from 'react';
import { NavLink } from 'react-router-dom';

const LeftNav = () => (
  <ul>
    <NavLink className="nav active" to="/inbox">Inbox &nbsp;&nbsp;&nbsp; <span className="notification"></span></NavLink> 
    <NavLink className="nav" to="/sent">Sent &nbsp;&nbsp;&nbsp; <span className="notification"></span></NavLink> 
    <NavLink className="nav" to="/trash">Trash &nbsp;&nbsp;&nbsp; <span className="notification"></span></NavLink> 
    <NavLink className="nav" to="/sign-out">Sign Out &nbsp;&nbsp;&nbsp; <span className="notification"></span></NavLink> 
</ul>
)

export default LeftNav;