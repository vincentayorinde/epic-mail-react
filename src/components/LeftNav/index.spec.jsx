
import React from 'react';
import { shallow } from 'enzyme';
import LeftNav from './index';

describe('LeftNav Component', () => {
  it('Should render without errors', () => {
    const component = shallow(<LeftNav />);
    expect(component.find('.nav').length).toBe(4);
    expect(component).toMatchSnapshot();
  });
});
