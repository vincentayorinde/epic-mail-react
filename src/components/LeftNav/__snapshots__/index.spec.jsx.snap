// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftNav Component Should render without errors 1`] = `
<ul>
  <NavLink
    className="nav active"
    to="/inbox"
  >
    Inbox     
    <span
      className="notification"
    />
  </NavLink>
  <NavLink
    className="nav"
    to="/sent"
  >
    Sent     
    <span
      className="notification"
    />
  </NavLink>
  <NavLink
    className="nav"
    to="/trash"
  >
    Trash     
    <span
      className="notification"
    />
  </NavLink>
  <NavLink
    className="nav"
    to="/sign-out"
  >
    Sign Out     
    <span
      className="notification"
    />
  </NavLink>
</ul>
`;
